rct229/ruleset_functions/baseline_systems/baseline_system_util.py

```diff
@@ -147,3 +151,104 @@ def find_exactly_one_fluid_loop(rmi, loop_id):
         loop_id,
         rmi,
     )
+
+
+def has_heating_system(rmi_b, hvac_b_id):
```

Comment:
Might this utility function be useful for non-baseline RMD's?  It might be better to keep it more general and drop the `_b`s.

Reply:
Great point! Fixed.